### DISCUSSION TITLE
get /device status selalu connected, tapi ketika post /message Device not authenticated

### DISCUSSION THREAD

**Discussion by akbarism** [kirimwa-id/forum#174] [Tanya Jawab] [answered]
saya mengecek apakah aplikasi saya harus rescan QR code lagi apa engga itu melalu api get device, kalo status != connected 
dan di whatsappnya session login sudah dihapus dari semua perangkat, tapi ketika get api /device status masih connected

response /device
```
{
    "id": "xiamoi-kentang",
    "created_at": "2021-10-07T10:53:23.273Z",
    "status": "connected",
    "connected_at": "2023-02-25T20:25:47.494Z",
    "disconnected_at": "2023-02-25T20:25:46.900Z",
    "disconnected_reason": "Logged out or invalid credentials"
}
```

reponse /message
```
message: 'Device is not authenticated, please rescan QR code.'
```

**Comment by udien-arrow** [comment]
saya juga sering seperti ini, apakah karena paket free ya?

**Reply by masbagusstwn** [reply]
yang sabar ya mas ..

**Reply by akbarism** [reply]
wkkw

**Reply by PojokHarapan** [reply]
Sama mas

**Reply by sorryjack** [reply]
saya juga mengalami

**Comment by irfanandratama** [comment]
Saya juga mengalami hal yang sama nih. Ketika GET /device hasilnya connected
`{"data":[{"id":"tab-galilea","created_at":"2022-12-26T04:01:24.739Z","status":"connected","connected_at":"2023-10-27T03:11:28.745Z","disconnected_at":"2023-10-27T03:11:28.432Z","disconnected_reason":"Logged out or invalid credentials"}],"meta":{"last_key":null}}`

**Comment by suharyadi13** [comment]
apakah ada yang sudah nemu solusi untuk masalah ini  ?

**Reply by udien-arrow** [reply]
manfaatin webhok type "device_status_changed"  kirim telegram jadi tau nomer lagi disconet atau konek

**Reply by akbarism** [reply, marked as answer]
Solusinya beli enterprise 😁😁😁
